.github/workflows/deploy_AppStore.yml

```diff
@@ -61,8 +61,8 @@ jobs:
       shell: bash
 
     - name: Build and Test
-      run: xcodebuild -workspace {workspace_name}.xcworkspace -scheme '{staging_scheme_name}' -destination 'platform=iOS Simulator,name=iPhone 12' clean test | xcpretty && exit ${PIPESTATUS[0]}
-
+      run: bundle exec fastlane build_and_test
+      
```

Comment:
please clean up line break indent.

Reply:
Updated in [13eb9c7](https://github.com/nimblehq/ios-templates/pull/170/commits/13eb9c733b22970de2430938b379219616453a99) 🙇‍♀️